Makefile

```diff
@@ -148,16 +148,18 @@ rundev:
 	cd scripts && ./run.sh
 
 run:
-	docker-compose -f docker/docker-compose.yml up
+	docker-compose -f docker/docker-compose.yml -f docker/aedes.yml up
```

Comment:
Aedes contains it's Redis. If I left them into docker-compose, then I would not be capable to replace them with additional `verne.yml`. While it would be possible to have generic `mqtt-adapter` that can be "casted" into Aedes by default and Verne by demand, it is impossible that other .yml removes services from previous (i.e. this Aedes Redis service).